src/Time/TimeSteppers/AdamsBashforthN.hpp

```diff
@@ -758,8 +758,9 @@ bool AdamsBashforthN::can_change_step_size(
   // evolution until the self-start history has been replaced with
   // "real" values.
   const evolution_less<Time> less{time_id.time_runs_forward()};
+  const evolution_less_equal<Time> less_equal{time_id.time_runs_forward()};
   return history.size() == 0 or
-         (less(history.back(), time_id.step_time()) and
+         (less_equal(history.back(), time_id.step_time()) and
```

Comment:
In the case of a length 1 history, the step size change should be permitted when the time step id is the same as the last in the history, because the step change now occurs before the advance time. The alternative, I suppose, would be to keep the same comparison operator but check the next time id instead of the current time id.

Reply:
This check is supposed to determine the "local direction of time flow".  If the two times are equal then something is wrong.  The goal of the check is to reject the case where, e.g., the previous steps were at times 1, 2, 3 and the next step will be at 0 (because the is_sorted check will pass there).

Reply:
I see. I think the way to keep this condition consistent with previous cases is then to just pass in the next time step (`Tags::Next<Tags::TimeStepId>`) instead of the current one (`Tags::TimeStepId`) and then I can revert the changes to the contents of the check.